packages/math-poolsimulator/src/gyro3/index.test.ts

```diff
@@ -0,0 +1,62 @@
+import { bnum, SubgraphPoolBase } from "@balancer-labs/sor";
+import { describe, expect, test } from "@jest/globals";
+
+import { numberToBigNumber } from "../conversions";
+import { AMM } from "../index";
+import poolsFromFile from "./fixtures/data.json";
+import { ExtendedGyro3 } from "./index";
+
+describe("Tests new Gyro 3CLP math function based on package other functions", () => {
+  const poolData = poolsFromFile["pool"];
+
+  const tokenIn = "USDC";
+  const tokenOut = "BUSD";
+
+  const amm = new AMM(
+    new ExtendedGyro3({
+      swapFee: poolData.swapFee,
+      totalShares: poolData.totalShares,
+      tokens: poolData.tokens,
+      tokensList: poolData.tokensList,
+      root3Alpha: poolData.root3Alpha,
+    }),
+  );
+
+  const pool = ExtendedGyro3.fromPool(poolData as SubgraphPoolBase);
+  const poolPairData = pool.parsePoolPairData(tokenIn, tokenOut);
+  const amount = 9104;
```

Comment:
Why did you choose this value?

Reply:
This almost all liquidity of the pool, right?

Reply:
uhum, I can add [this ](https://linear.app/bleu-llc/issue/BAL-607/test-the-spot-price-for-amounts-equal-to-25-50-75-95-percent-of-the)too

Reply:
done